README.md

```diff
@@ -19,11 +19,17 @@ And then execute:
     $ bundle
 
 
+## Usage
 ## Usage
 ```ruby
-# Capfile
-require 'capistrano/sidekiq'
-require 'capistrano/sidekiq/monit' #to require monit tasks # Only for capistrano3
+    # Capfile
+
+    require 'capistrano/sidekiq'
+    install_plugin Capistrano::Sidekiq  # Default sidekiq tasks
+    # Then select your service manager
```

Comment:
Please include monit here.

Reply:
@seuros please include monit here.

Reply:
Readme is totaly outdated, i will do that once the gem is full working in this branch.

You can open PR against this branch if you can help. I will merge